scanning/oscap-debug/sysctl-oval-eval-parallel.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+
+import time
+import subprocess
+import concurrent.futures
+
+from lib import util, results
+
+
+def run(fn):
+    handle = util.subprocess_Popen(['oscap', 'oval', 'eval', fn])
+    try:
+        rc = handle.wait(OSCAP_TIMEOUT)
+        return handle.pid, rc
+    except subprocess.TimeoutExpired:
+        return handle.pid, -1
+
+
+start_time = time.monotonic()
+
+# sysctl ovals only take about 1 second
+OSCAP_TIMEOUT = 5
```

Comment:
Probably move the constant to the top, above `def run()`, so the reader notices its definition before use in the function.

Reply:
Yeah.